Manifold/Term+Construction.swift

```diff
@@ -56,6 +56,10 @@ extension Term {
 		return .Embedded(A.self, (==) as (A.Type, A.Type) -> Bool, .Type)
 	}
 
+	public static var Implicit: Term {
+		return nil
+	}
```

Comment:
You can’t always use the `NilLiteralConvertible` instance (e.g. in `someTerm[nil]`, tho it’s unclear why), so this static variable is available.